Assets/NuGet.config

```diff
@@ -12,4 +12,24 @@
       <add key="repositoryPath" value="./Packages" />
       <add key="DefaultPushSource" value="http://www.nuget.org/api/v2/" />
    </config>
+   <nativeRuntimesMapping>
```

Comment:
While I don't mind having these be data driven I don't think the nuget.config is the right place for them. Can you move them to a scriptable object, separate file, or simply hard code the mapping?

Reply:
Yeah realised that the other day, I plan to put it into a file in the `NuGet` folder and use something like Yaml/Json to make parsing easier

Reply:
Reworked to use "settings.json" inside the "NuGet" folder